### PR TITLE
GUI-2810 Instance Selector for elb wizard

### PR DESCRIPTION
**Pull request by dkavanagh**
https://eucalyptus.atlassian.net/browse/GUI-2810
Refactoring of magic search filtering code into service required import added to all landing pages and a few more.

### CODE REVIEW

File: eucaconsole/static/js/pages/elb-wizard/instances.js [hunk 1]
```diff
@@ -7,16 +7,29 @@
...
-        ELBWizardService.next({});
-    };
-}]);
+.controller('InstancesController', ['$scope', '$routeParams', 'InstancesService', 'eucaHandleError',
+            'ELBWizardService',
+    function ($scope, $routeParams, InstancesService, eucaHandleError, ELBWizardService) {
+        var vm = this;
+        vm.vpcNetwork = 'None';
+        vm.availabilityZones = [];
+        vm.availabilityZoneChoices = [{id:'one', label:'one'}];
```
Comment (kamalgill) [inline eucaconsole/static/js/pages/elb-wizard/instances.js]: Are the availability zone or subnet choices meant to be hard-coded here?

File: eucaconsole/static/js/pages/landingpage.js [hunk 2]
```diff
@@ -25,7 +25,8 @@ angular.module('LandingPage', ['CustomFilters', 'ngSanitize', 'MagicSearch', 'Ex
             }
         };
     })
-    .controller('ItemsCtrl', function ($scope, $http, $timeout, $sanitize, $location, lpModelService) {
+    .controller('ItemsCtrl', ['$scope', '$http', '$timeout', '$sanitize', '$location', 'lpModelService', 'MagicSearchFilterService',
```
Comment (kamalgill) [inline eucaconsole/static/js/pages/landingpage.js]: Recommend we name lpModelService as LandingPageModelService

File: eucaconsole/templates/elbs/instance-selector.pt [hunk 3]
```diff
@@ -0,0 +1,73 @@
+<div>
```
Comment (kamalgill) [inline eucaconsole/templates/elbs/instance-selector.pt]: Need an `i18n:domain="eucaconsole"` attribute here

Comment (dkavanagh) [inline eucaconsole/templates/elbs/instance-selector.pt]: yes, will do

**Pull request merged**